types/printer/index.d.ts

```diff
@@ -0,0 +1,67 @@
+// Type definitions for printer 0.4
+// Project: http://github.com/tojocky/node-printer
+// Definitions by: Christos Panagiotakopoulos <https://github.com/chrispanag>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export as namespace Printer;
```

Comment:
```suggestion

export as namespace Printer;
```
It doesn't look like UMD module for me, so that export is probably not required.
It's purely NodeJS focused module, is there an UMD module for that package?
https://www.typescriptlang.org/docs/handbook/modules.html#umd-modules

Reply:
You are right, this is a purely NodeJS module.